ExampleGallery/Effects/VariableVideoEffect.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Windows.Foundation.Collections;
```

Comment:
I will add this when rewriting the class with the effect name instead of the concept name.